sphinx/tutorial/cylc/furthertopics/family-triggers.rst

```diff
@@ -0,0 +1,184 @@
+Family triggers
+===============
+
+This tutorial walks you through using family :term:`triggers <task trigger>`.
+
+Family triggers allow you to build dependencies based on sets of tasks
+sharing the same namespace, referred to as
+:ref:`families <tutorial-cylc-families>`.
+
+
+Purpose
+-------
+
+Family triggers are used to specify a condition to be met by a set of tasks
+in a particular :term:`family` that will :term:`trigger <task trigger>`
+the next task(s) in the :term:`graph`.
+
+Like regular task triggers, family triggers can depend on the success or
+failure of tasks. However, because a family can contain multiple tasks, you
+need to specify whether you are concerned with all or any of the tasks in
+that family reaching the desired state.
+
+
+Example
+-------
+
+Create a new suite called tutorial-family-triggers::
+
+   mkdir ~/cylc-run/tutorial-family-triggers
+   cd ~/cylc-run/tutorial-family-triggers
+
+Paste the following configuration into the ``suite.rc`` file:
+
+.. code-block:: cylc
+
+   [cylc]
+       UTC mode = True # Ignore DST
+   [scheduling]
+       [[dependencies]]
+           graph = visit_mine => MINERS
+   [runtime]
+       [[visit_mine]]
+           script = sleep 5; echo 'off to work we go'
+
+       [[MINERS]]
+           script = """
+   sleep 5;
+   if (($RANDOM % 2)); then
+       echo 'Diamonds!'; true;
+   else
+       echo 'Nothing...'; false;
+   fi
+   """
+       [[doc, grumpy, sleepy, happy, bashful, sneezy, dopey]]
+           inherit = MINERS
+
+You have now created a suite that:
+
+* Has a ``visit_mine`` task that sleeps for 5 seconds then outputs a
+  message.
+* Contains a ``MINERS`` family with a command in it that randomly succeeds
+  or fails.
+* Has 7 tasks that inherit from the ``MINERS`` family.
+
+Open the ``cylc gui`` then run the suite by pressing the "play" button
+(top left hand corner) then clicking :guilabel:`Start`::
```

Comment:
The ':guilabel:' formatting of the 'Start button' is inconsistent with the rest of the documentation (including earlier in this very sentence!) where buttons / other GUI items are instead referred to in plain text using double quotation marks. I strongly recommend changing to the standard formatting as it is otherwise confusing.

Reply:
> including earlier in this very sentence!

So this is actually purposeful, perhaps just me being pedantic but the play button is marked with a symbol not text (so there is no "play" button whereas there is a ⏯ button). On the other hand there is actually a button labelled "Start".

Reply:
Ah, makes sense - forgive my misapprehension. In that case once we (eventually) go through and 'guilabel' the text-based buttons it should all be very clear and nicely formatted.